v2/pkg/templates/templates.go

```diff
@@ -45,10 +46,16 @@ type Template struct {
 	Info model.Info `yaml:"info" json:"info" jsonschema:"title=info for the template,description=Info contains metadata for the template"`
 	// description: |
 	//   Requests contains the http request to make in the template.
+	//   WARNING: 'requests' will be deprecated and will be removed in a future release. Please use 'http' instead.
 	// examples:
 	//   - value: exampleNormalHTTPRequest
 	RequestsHTTP []*http.Request `yaml:"requests,omitempty" json:"requests,omitempty" jsonschema:"title=http requests to make,description=HTTP requests to make for the template"`
 	// description: |
+	//   HTTP contains the http request to make in the template.
```

Comment:
@ShubhamRasal , I think we can mark this field as deprecated (ref:https://zchee.github.io/golang-wiki/Deprecated/)  by doing this any integration/imports built on top of this would be detected and warned by golangci and other tools